_overviews/toolkit/sttp-make-request.md

```diff
@@ -0,0 +1,40 @@
+---
+title: How to make an HTTP request?
+type: section
+description: How to make any HTTP request with Scala Toolkit.
+num: 13
+previous-page: sttp-intro
+next-page: sttp-variable-urls
+---
+
+{% include markdown.html path="_markdown/install-sttp.md" %}
+
+## Make a request
+The simplest way to make a request with Scala Toolkit's http client library, sttp, is to use the `SimpleHttpClient` instance. 
+You need to import a few members from the library, and then write the code that makes the request:
+
+```scala
+import sttp.client3.{SimpleHttpClient, UriContext, basicRequest}
+
+val client = SimpleHttpClient() // Create the instance of SimpleHttpClient
+val request = basicRequest.get(uri"https://httpbin.org/get") // Construct a get request to an example service - https://httpbin.org/get
+val response = client.send(request) // send the request and get the response
+println(response.body) // print the body of the response
+```
+
+## Other request types
+You can swap `basicRequest.get` with any HTTP method you want. For example:
+```scala
+val request = basicRequest.post(uri"https://httpbin.org/get")
+```
+Will make a `POST` request.
+
+## The `basicRequest`
+Sttp library takes an approach of starting with a simple request prototype, the `basicRequest`, then modifying it to suit your needs. 
+The `basicRequest` is already set up with some [basic headers](https://sttp.softwaremill.com/en/latest/requests/basics.html#initial-requests) - all you need to do is provide the address and the method you want to preform with `get` function. 
+Other functions are available for different HTTP methods. 
+
+## The `uri` operator before website address
```

Comment:
[nitpick] I'm not sure if that's an operator in the Scala sense? I think the proper name is "interpolator", but not sure if that would be any easier to understand for beginners

Reply:
I thought about it a bit, and interpolator may be a word that newcomers won't understand.

Reply:
ok, I'm fine with that if that's a consciously made simplification :)

Reply:
I agree with @adamw here. I don't think using the word `operator` instead of `interpolator` is correct. An interpolator is a specific thing in Scala with a specific syntax, and an operator is something different.

While I agree we should try to make the explanation as beginner friendly as possible, I think the work `interpolator` is okay here. It becomes clear as soon as we say that an interpolator can interpolate values.